README.md

```diff
@@ -271,13 +271,44 @@ To try out Carbon, you can use the Carbon explorer to interpret Carbon code and
 print its output. You can try it out immediately at
 [compiler-explorer.com](http://carbon.compiler-explorer.com/).
 
-Because Carbon is an early, experimental project we don't yet have releases you
-can download and try out locally, you'll instead need to build any tools
-yourself from source. We expect to have packaged releases you can try out when
-we reach our
+Carbon is an early, experimental project, and so we only have very experimental
+nightly releases of the Carbon toolchain available to download, and only on
+limited platforms. If you are using a recent Ubuntu Linux or similar (Debian,
+WSL, etc.), you can try these out by going to our
+[releases](https://github.com/carbon-language/carbon-lang/releases) page and
+download the latest nightly toolchain tar file:
+`carbon_toolchain-v0.0.0-0.nightly.YYYY.MM.DD.tar.gz`. Then you can try it out:
+
+```shell
+# A variable with the specific nightly version:
+VERSION="v0.0.0-0.nightly.YYYY.MM.DD"
+
+# Unpack the toolchain:
+tar -xvf carbon_toolchain-${VERSION}.tar.gz
+
+# Create a simple Carbon source file:
+echo "fn Run() { Print(42); }" > forty_two.carbon
+
+# Compile to an object file:
+./carbon_toolchain-${VERSION}/bin/carbon compile \
+  --output=forty_two.o forty_two.carbon
+
+# Link to an executable:
+./carbon_toolchain-${VERSION}/bin/carbon link \
```

Comment:
The latter instructions include clang, libc++, and lld. Do we need any of that in order to link?

Reply:
I don't *think* so. Definitely don't need clang or lld.

I'll try this out with a fresh Ubuntu cloud image just to see what if anything might be needed there.

@KateGregory -- anything we should include for WSL?

Reply:
I've added the minimal necessary `sudo apt install` based on my testing right before linking (it isn't needed until linking).